pom.xml

```diff
@@ -442,7 +450,7 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-compiler-plugin</artifactId>
-					<version>3.1</version>
+					<version>3.8.0</version>
```

Comment:
Anything we might need to consider before bumping these versions? Can they be transitive in some way that might lead to other issues?

Reply:
I don't think these versions are transitive. Anything contained in either `<build>` or `<pluginManagement>` should be contained to only modules specified by this parent.